src/bot.rs

```diff
@@ -221,14 +217,7 @@ impl Bot {
                         "Chat {chat_id} is not in the database, adding it as chat with type store"
                     );
                         db::set_chat_type(conn, chat_id, ChatType::Store).await?;
-                        let msg = send_webxdc(
-                            context,
-                            &state,
-                            chat_id,
-                            Webxdc::Store,
-                            Some(store_message()),
-                        )
-                        .await?;
+                        let msg = send_store_xdc(context, &state, chat_id, true).await?;
```

Comment:
ah much better name -- i assume it's the frontend xdc, right?

Reply:
yes